src/test/java/io/debezium/connector/jdbc/JdbcSinkConnectorToMySQLIT.java

```diff
@@ -0,0 +1,94 @@
+
+import static io.debezium.connector.jdbc.SourcePipelineInvocationContextProvider.DATABASE_CONTAINER_NAME;
+import static io.debezium.connector.jdbc.SourcePipelineInvocationContextProvider.KAFKA_CONNECT_CONTAINER_NAME;
+import static io.debezium.connector.jdbc.SourcePipelineInvocationContextProvider.KAFKA_CONTAINER_NAME;
+import static io.debezium.connector.jdbc.SourcePipelineInvocationContextProvider.getNetwork;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.Properties;
+import java.util.regex.Pattern;
+
+import org.apache.kafka.clients.consumer.ConsumerConfig;
+import org.apache.kafka.clients.consumer.KafkaConsumer;
+import org.apache.kafka.connect.sink.SinkRecord;
+import org.junit.Assert;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.TestTemplate;
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.containers.JdbcDatabaseContainer;
+import org.testcontainers.containers.KafkaContainer;
+import org.testcontainers.containers.MySQLContainer;
+import org.testcontainers.utility.DockerImageName;
+
+import io.debezium.testing.testcontainers.Connector;
+import io.debezium.testing.testcontainers.ConnectorConfiguration;
+import io.debezium.testing.testcontainers.DebeziumContainer;
+
+public class JdbcSinkConnectorToMySQLIT extends AbstractPipelineIT {
+    private static DockerImageName MYSQL_IMAGE = DockerImageName.parse("mysql:8.0");
+
+    private static MySQLContainer<?> sinkDatabaseContainer = new MySQLContainer<>(MYSQL_IMAGE).withNetwork(getNetwork())
+            .withDatabaseName("test");
+
+    @BeforeEach
+    void before() {
+        sinkDatabaseContainer.start();
+    }
+
+    @AfterEach
+    void after() {
+        sinkDatabaseContainer.stop();
+    }
+
+    @TestTemplate
+    void createSimplePipelineFromSourceToSink(Map<String, GenericContainer<?>> containerMap) throws InterruptedException {
+        KafkaContainer kafkaContainer = (KafkaContainer) containerMap.get(KAFKA_CONTAINER_NAME);
+        DebeziumContainer kafkaConnectContainer = (DebeziumContainer) containerMap.get(KAFKA_CONNECT_CONTAINER_NAME);
+        JdbcDatabaseContainer sourceDatabaseContainer = (JdbcDatabaseContainer) containerMap.get(DATABASE_CONTAINER_NAME);
+
+        // Register SourceConnector
+        String sourceConnectorName = String.format("source-%s", getDatabaseName(sourceDatabaseContainer));
+        ConnectorConfiguration sourceConnectorConfiguration = getCommonSourceConnectorConfiguration(sourceDatabaseContainer);
+        kafkaConnectContainer.registerConnector(
+                sourceConnectorName,
+                sourceConnectorConfiguration);
+        Assert.assertTrue(
+                String.format("Source connector [%s] is not registered", sourceConnectorName),
+                kafkaConnectContainer.isConnectorConfigured(sourceConnectorName));
+
+        Thread.sleep(5000);
```

Comment:
Let's rather use Awaitility (see other tests in Debezium for an example) instead of any unconditional waits.